tests/models_test/inprocess_test/models_inprocessing_test.py

```diff
@@ -178,17 +194,17 @@ def kamishima_teardown() -> Generator[None, None, None]:
 
     This has to be done with a fixture because otherwise it will not happen when the test fails.
     """
-    yield
+    yield Kamishima()
     print("teardown Kamishima")
     Kamishima().remove()  # delete the downloaded code
 
 
 @pytest.mark.slow
-def test_kamishima(toy_train_test: TrainTestPair, kamishima_teardown: None) -> None:
+def test_kamishima(toy_train_test: TrainTestPair, kamishima: Kamishima) -> None:
```

Comment:
you changed the name of the function argument, but not the name of the fixture

Reply:
changed, but this test doesn't work on my machine... googling the error message says something about python2

Reply:
Yeah, don't worry about it. It does work on hydra